src/py/mains/consensus.py

```diff
@@ -206,9 +236,9 @@ def Start():
 
     K = 8
     config = args.min_cov, K, args.local_match_count_window, args.local_match_count_threshold,\
-             args.max_n_read, args.min_idt, args.edge_tolerance, args.trim_size
+             args.max_n_read, args.min_idt, args.edge_tolerance, args.trim_size, args.min_cov_aln, args.max_cov_aln
     # TODO: pass config object, not tuple, so we can add fields
```

Comment:
Haha! I guess I should have done this TODO a long time ago. It would be helpful here.